v22.2/hasura-getting-started.md

```diff
@@ -0,0 +1,193 @@
+    ~~~
+1. Click **Add**.
+
+### Create a Data Source in Hasura Cloud
+
+1. In the overview page in Hasura Cloud, select **Projects**, find the project you created earlier, and click **Launch Console**.
+1. Click **Data** in the console header to open the **Data Manager**.
+1. Click **Connect Database**.
+1. In the **Connect Existing Database** tab configure the connection to your cluster.
+    1. Under **Database Display Name** type a name for your connection. Your cluster name is a good option. For example, `fake-puppy`.
+    1. Set **Data Source Driver** to **CockroachDB**.
+    1. Select **Environment Variable**, then enter `CRDB_URL` in the **Environment Variable** input box.
+        <img src="{{ 'images/v22.2/hasura-data-source.png' | relative_url }}" alt="Adding the connection string to the Hasura Data Manager" style="border:1px solid #eee;max-width:100%" />
+   1. Expand **SSL Certificates Settings** and set **SSL Mode** to `verify-full`.
+   1. Note that **SSL Root Certificate** is set to the `SSL_ROOT_CERT` environment variable you configured earlier.
+1. Click **Connect Database**.
+
+## Add the Hasura Cloud network to your cluster allowlist
+
+Your {{ site.data.products.dedicated }} cluster needs to be configured to [allow incoming client connections](../cockroachcloud/network-authorization.html#ip-allowlisting) from Hasura Cloud.
+
+1. In the Hasura Cloud overview page select **Projects**, then click the **Config** icon for your project.
+
+    <img src="{{ 'images/v22.2/hasura-project-config.png' | relative_url }}" alt="Click the project Config icon" style="border:1px solid #eee;max-width:100%" />
+1. In the **General** section, copy the IP address listed under **Hasura Cloud IP**.
+1. In the CockroachDB Cloud console select your cluster, click **Networking**, then **Add Network**.
+1. In the **Add Network** dialog, set **Network name** to `Hasura Cloud`.
+1. Under **Network**, select **New Network**. Paste the IP address of your Hasura Cloud instance you copied earlier.
+1. Check **CockroachDB Client to access the databases**, then click **Apply**.
+
+## Create a table in your CockroachDB cluster
+
+1. Connect to your cluster using the SQL client.
+   
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach sql --url "{connection string}"
+    ~~~
+
+    Where `{connection string}` is the connection string for your cluster you copied earlier.
+
+1. In the SQL client, create an `accounts` table.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE TABLE IF NOT EXISTS accounts (
+      id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+      balance INT8
+    );
+    ~~~
+
+1. Insert some data into the `accounts` table.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    UPSERT INTO accounts (balance) VALUES (1000), (250);
+    ~~~
+
+This will create two accounts, one with $1000 and another with $250.
+
+## Track the <code>accounts</code> table in the Hasura Cloud Data Manager
```

Comment:
as of this writing I was able to do all the steps up to here but my Hasura conn is failing (see above)